README.md

```diff
@@ -87,18 +87,23 @@ Inside your `AppDelegate.application(_, didFinishLaunchingWithOptions:)` impleme
     }
 ```
 
-Implement the two required `StreemDelegate` methods:
+Implement the required `StreemDelegate` methods:
 
 ```swift
     public func currentUserDidChange(user: StreemUser?) {
-        // as necessary, update your stored and/or displayed `user.name` and `user.id`
+        // As necessary, update your stored and/or displayed `user.name` and `user.id`
     }
     
     public func func measurementUnitDidChange(measurementUnit: UnitLength) {
         // Your app might wish to copy this value to your app preferences,
         // so that when starting future sessions you can restore it
         // (by setting `Streem.sharedInstance.measurementUnit`).
     }
+    
+    public func helpRequested() {
+        // This function relates to an upcoming Streem feature.
+        // For now, you can simply provide a do-nothing stub here.
+    }
```

Comment:
This is what I get for taking so long to release this updated SDK...

Reply:
Can we not make it an optional delegate method?